galeracluster/source/training/courses/index.rst

```diff
@@ -150,13 +150,26 @@ Note that Hands On Labs are denoted with HOL.
 
 - The cost of training is 1800 € / 2100 USD for 2 days.
 
-- Early bird price before 26th of August 2021 1500 €/1750 USD for 2 days.
+- Early bird price 1500 €/1750 USD for 2 days applies.
 
 - Volume discount for trainees from the same company: 4 pays for 3, 5 pays for 4 etc.
 
 - `Sign Up For Training <http://www2.galeracluster.com/l/38852/2020-09-18/9kqcz6>`_
 
 
+   .. only:: html
+
+          .. image:: ../../images/training.jpg
```

Comment:
I think this is going to fail. it needs to be ../images/training.jpg -- can you please check @kirjaamo